packages/appstore/src/components/add-more-wallets/wallet-add-card.tsx

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import { TWalletInfo } from 'Types';
 import { Text, WalletCard } from '@deriv/components';
-import { useCurrencyConfig } from '@deriv/hooks';
+import { useCurrencyConfig } from '@deriv/api';
```

Comment:
@arshad-rao-deriv please revert it back since these hooks from api package are slighly different from hooks package

Reply:
Updated.